tests/e2e/workloads/couchbase/test_couchbase.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import E2ETest, workloads, ignore_leftovers
+from ocs_ci.ocs.couchbase import CouchBase
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def couchbase(request):
+
+    couchbase = CouchBase()
+
+    def teardown():
+        couchbase.teardown()
+    request.addfinalizer(teardown)
+    return couchbase
+
+
+@workloads
+@ignore_leftovers
+@pytest.mark.polarion_id("OCS-807")
+class TestCouchBaseWorkload(E2ETest):
+    """
+    Deploy an CouchBase workload using operator
+    """
+    def test_cb_workload_simple(self, couchbase):
+        couchbase.create_couchbase_worker(replicas=5)
+        couchbase.run_workload(replicas=5)
+        couchbase.analyze_run()
```

Comment:
I see sanity_check() in pillowfight. Do you think we should add that at the end of the test?

Reply:
sanity_check()  is a part of analyze_run()